src/modules/homepage.js

```diff
@@ -0,0 +1,21 @@
+import { Link } from 'react-router-dom';
+
+const Homepage = () => (
+  <section className="home-page">
+    <img src="https://img.icons8.com/external-icons-smashing-stocks/68/000000/external-Currencies-stock-market-icons-smashing-stocks.png" alt="App logo" />
+    <div>
+      <h1>Welcome to Currex</h1>
+    </div>
+    <Link to="continents">
+      <button
+        type="button"
+        name="proceed"
+        id="proceed"
+      >
+        Proceed
+      </button>
+    </Link>
+  </section>
+);
+
+export default Homepage;
```

Comment:
- [ ] I am puuting this comment here so you can understand what I am saying, according to the project requirement, a user should be able to search for a currency, so you will have to use a filter method in your redux to do this, kindly add this as it is the only thing left for you to finish, you have a wonderful project already :clap:

Reply:
Hello 
please note that I have implemented the search bar on the third page as that is more viable